raytracing/matrix.py

```diff
@@ -746,7 +751,14 @@ def displayHalfHeight(self):
             halfHeight = self.apertureDiameter / 2.0  # real half height
         return halfHeight
 
+    def axesScale(self, axes):
+        xlims, ylims = axes.get_xbound(), axes.get_ybound()
+        xScale, yScale = xlims[1]-xlims[0], ylims[1]-ylims[0]
+        return xScale, yScale
+
     def axesToDataScaling(self, axes):
+        raise DeprecationWarning("Use .axesScale() instead")
```

Comment:
See comment above regarding keeping the reference frames explicit in the function names.